docs_app/content/tutorial/4-what-is-an-operator.md

```diff
@@ -0,0 +1,71 @@
+# 4. What Is An Operator?
+
+## TLDR:
+
+- Observables are a container for a collection of values over time.
```

Comment:
Is there another word that we can use instead of "collection"? One thing that confuses developers new to RxJS is that it doesn't retain a collection of data and you can't just get the values out of an Observable. (Except for the BehaviorSubject.value, which IMHO shouldn't be used as it often is.) Could we use "stream" here? An Observable is not a stream, but an Observable provides a stream of values over time? Or some other word other than collection?